mkdocs.yml

```diff
@@ -16,7 +16,8 @@ pages:
 - ['docs/getting-started.md', 'Documentation', 'Getting Started']
 - ['docs/schema.md', 'Documentation', 'Working with table schemas']
 #- ['docs/usage.md', 'Documentation', 'Usage']
-- ['docs/models.md', 'Documentation', 'Models']
+- ['docs/models-definition.md', 'Documentation', 'Models - Definition']
+- ['docs/models-usage.md', 'Documentation', 'Models - Usage']
```

Comment:
It adds 3+ levels now ;) I just need to add support to all the themes. One down (see comment on the PR) a few to go!